app/serializers/rule_serializer.rb

```diff
@@ -3,6 +3,9 @@
 # JSON API serialization for an OpenSCAP Rule
 class RuleSerializer
   include FastJsonapi::ObjectSerializer
-  attributes :created_at, :updated_at, :ref_id, :title, :rationale,
-             :description, :severity, :slug
```

Comment:
created_at/updated_at isn't returned by any other serializers. IMO we could return it for all resources or none.